Formula/babel.rb

```diff
@@ -4,8 +4,8 @@
 class Babel < Formula
   desc "Compiler for writing next generation JavaScript"
   homepage "https://babeljs.io/"
-  url "https://registry.npmjs.org/@babel/cli/-/cli-7.4.4.tgz"
```

Comment:
`https://registry.npmjs.org/@babel/cli/-/cli-7.4.5.tgz` seems not available for downloading yet.

Reply:
![image](https://user-images.githubusercontent.com/1580956/58121792-31c93b00-7bd6-11e9-95ea-441da42f6137.png)